### ISSUE TITLE
strange format of long tuple

### ISSUE THREAD

**Issue by liuchong** [rust-lang-nursery/rustfmt#933]
before format
```rust
fn main() {
    let (aaaaaaa, bbbbbbb, ccccccc, ddddddd, eeeeeee, fffffff, ggggggg) = (1,2,3,4,5,6,7);
}
```
after format
```rust
fn main() {
    let (aaaaaaa, bbbbbbb, ccccccc, ddddddd, eeeeeee, fffffff, ggggggg) = (1,
                                                                           2,
                                                                           3,
                                                                           4,
                                                                           5,
                                                                           6,
                                                                           7);
}
```
I think maybe like this if better, how could I achieve this or some style which is similar to this?
``` rust
    let (aaaaaaa, bbbbbbb, ccccccc, ddddddd, eeeeeee, fffffff, ggggggg) =
        (1,2,3,4,5,6,7);
```

Thanks!

**Comment by marcusklaas** [comment]
This is an important issue that we will fix before our 1.0 release. It has already been filed before in a different form: https://github.com/rust-lang-nursery/rustfmt/issues/497. I'll close this one for being a duplicate.

**Comment by liuchong** [comment]
Thanks!
